presto-main/src/test/java/io/prestosql/sql/planner/TestLiteralEncoder.java

```diff
@@ -90,28 +98,154 @@
             ImmutableSet.of());
 
     @Test
-    public void testEncode()
+    public void testEncodeNull()
     {
-        assertEncodeCaseInsensitively(utf8Slice("hello"), VARBINARY, literalVarbinary("hello".getBytes(UTF_8)));
         assertEncode(null, UNKNOWN, "null");
         assertEncode(null, BIGINT, "CAST(null AS bigint)");
```

Comment:
nit: I'd like the null timestamp and timestamptz changes to also go here. Or move the BIGINT case into `testEncodeBigint`.

Reply:
rearranged as requested